plugin/commands.py

```diff
@@ -110,6 +110,32 @@ def _get_fold_section_names(window: sublime.Window) -> set[str]:  # type: ignore
     return names
 
 
+def _format_patch_changes(changes: dict) -> str:
+    """Return a Markdown representation of patch *changes*."""
+
+    body = ''
+
+    if not isinstance(changes, dict):
+        return body
```

Comment:
```suggestion
        try:
            dump = json.dumps(changes, indent=2)
            return f'```json\n{dump}\n```\n\n'
        except Exception:
            dump = str(changes)
            return f'```\n{dump}\n```\n\n'
```